app/components/support/crate-report-form.hbs

```diff
@@ -0,0 +1,94 @@
+<form
+  local-class="report-form"
+  data-testid="crate-report-form"
+  ...attributes
+  {{on "submit" (prevent-default this.submit)}}
+>
+  <h2>Report A Crate</h2>
+
+  <fieldset local-class="form-group" data-testid="fieldset-name">
+    {{#let (unique-id) as |id|}}
+      <label for={{id}} local-class="form-group-name">
+        Crate name
+        {{#if this.loadCrateTask.isRunning}}
+          <LoadingSpinner />
+        {{/if}}
+      </label>
+      <Input
+        id={{id}}
+        @type="text"
+        @value={{this.name}}
+        disabled={{this.loadCrateTask.isRunning}}
+        autocomplete="off"
+        aria-required="true"
+        aria-invalid={{if this.nameInvalid "true" "false"}}
+        local-class="name-input"
+        data-testid="name-input"
+        {{auto-focus}}
+        {{on "input" this.resetNameValidation}}
+      />
+      {{#if this.nameInvalid}}
+        <div local-class="form-group-error" data-testid="name-invalid">
+          Please specify an existing crate.
+        </div>
+      {{/if}}
```

Comment:
since we ultimately generate an email, where the user can change the name again I'm wondering if the extra name validation might be overkill here 😅

Reply:
Fair enough! It might be overkill. I implemented it primarily to prevent typos. Do you suggest simply not checking for existing names?

Reply:
yeah, I expect most users to go through the report crate button anyway, which auto-fills the field and thus shouldn't have any typos